snuba/admin/clickhouse/nodes.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import annotations
+
+from typing import Sequence, TypedDict
+
+from snuba.datasets.schemas.tables import TableSchema
+from snuba.datasets.storages import StorageKey
+from snuba.datasets.storages.factory import STORAGES, get_storage
+
+Storage = TypedDict(
+    "Storage",
+    {"storage_name": str, "local_table_name": str, "local_nodes": Sequence[str]},
+)
+
+
+def _get_local_table_name(storage_key: StorageKey) -> str:
+    schema = get_storage(storage_key).get_schema()
+    assert isinstance(schema, TableSchema)
+    return schema.get_table_name()
+
+
+def _get_local_nodes(storage_key: StorageKey) -> Sequence[str]:
+    storage = get_storage(storage_key)
+    return [
+        f"{node.host_name}:{node.port}"
+        for node in storage.get_cluster().get_local_nodes()
```

Comment:
This will fail in production as we have clusters that are not single node but that have no cluster name set 
https://github.com/getsentry/snuba/blob/master/snuba/clusters/cluster.py#L272

Reply:
I forgot about that, thanks